dependencies/jessie/hammer_cli/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-hammer-cli (0.5.1-1) stable; urgency=low
+
+  * 0.5.1 release
+
+ -- Tomas Strachota <tstrachota@redhat.com>  Mon, 15 Dev 2015 16:04:13 +0100
```

Comment:
Tuesday today, and this should read "Dec" (error below I guess).  You can use `scripts/update_package.rb --name hammer_cli --version 0.5.1` to auto-generate everything.